CompanyEmployees.Core.Domain/Exceptions/AttendanceAlreadyExistsException.cs

```diff
@@ -0,0 +1,9 @@
+namespace CompanyEmployees.Core.Domain.Exceptions;
+
+public sealed class AttendanceAlreadyExistsException : BadRequestException
```

Comment:
Consider adding an optional `DateTime date` parameter to enhance clarity in the message, especially if this exception is raised for a specific date other than today. This allows better future extensibility and debugging.

**Suggested change**
```suggestion
public sealed class AttendanceAlreadyExistsException : BadRequestException
{
    public AttendanceAlreadyExistsException(Guid employeeId, DateTime date)
        : base($"Employee with ID {employeeId} has already checked in on {date:yyyy-MM-dd}.")
    {
    }
}
```